lib/prayermate_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module PrayerMateApi
-  VERSION = "0.2.0"
-end
+  VERSION = "0.2.1"
```

Comment:
Since this is a breaking change as it stands, you need to make this "0.3.0"

Reply:
Yeah not sure why this is coming up. The PR seems to be picking up stuff from the previous PR where I bumped it to 0.2.1. Definitely needs to be 0.3.0. I was thinking of rolling in the publish as draft input_feed stuff too.